src/components/FilterList/FilterList.tsx

```diff
@@ -1,56 +1,19 @@
 import { PropsWithChildren } from 'react';
 import { Stack } from '@mantine/core';
 
-import { useWindowSize } from '@/windowmanagement/Window/hooks';
-
 import { SearchResults } from './SearchResults/SearchResults';
 import { FilterListFavorites } from './FilterListFavorites';
 import { FilterListInputField } from './FilterListInputField';
 import { FilterListProvider } from './FilterListProvider';
 
-interface FuncHeightProps {
-  heightFunc?: (height: number) => number;
-  heightPercent?: never;
-  height?: never;
-}
-
-interface AbsoluteHeightProps {
-  height?: number | string;
-  heightPercent?: never;
-  heightFunc?: never;
-}
-
-interface RelativeHeightProps {
-  heightPercent?: number;
-  height?: never;
-  heightFunc?: never;
-}
-
-type HeightProps = FuncHeightProps | AbsoluteHeightProps | RelativeHeightProps;
-
-interface BaseProps extends PropsWithChildren {
+interface Props extends PropsWithChildren {
   isLoading?: boolean;
+  height?: number | string;
 }
 
-type Props = HeightProps & BaseProps;
-
-export function FilterList({
-  height,
-  heightFunc,
-  heightPercent,
-  isLoading,
-  children
-}: Props) {
-  const { height: windowHeight } = useWindowSize();
-
-  const calculatedHeight =
-    height ??
-    (heightPercent && windowHeight * (heightPercent / 100.0)) ??
-    (heightFunc && heightFunc(windowHeight)) ??
-    '100%';
-
+export function FilterList({ height = '100%', isLoading, children }: Props) {
   return (
-    <Stack style={{ height: calculatedHeight }}>
+    <Stack style={{ height }}>
```

Comment:
```suggestion
    <Stack style={{ height }} gap={'xs'}>
```
:) ? Have been doing this change in some other PR I think, to reduce the cap between the filterlist parts. Would be nice to do it here too to reduce conflicting code